modules/scripts/genindex.py

```diff
@@ -115,6 +120,7 @@ def main():
     if len(subprojects) != len(sub_docpaths):
         raise ValueError("Invalid arguments")
     content = get_index(args.root, subprojects, sub_docpaths)
+    content = add_javadoc(content)
```

Comment:
ensure that the `get_index` method returns the entire index. Call `add_javadoc` from `get_index`